packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -59,7 +63,10 @@ function SelectionMesh(props: Props) {
       if (!startPoint) {
         return;
       }
-      const point = worldToData(evt.unprojectedPoint);
+      const boundedX = clamp(evt.unprojectedPoint.x, fov.left, fov.right);
+      const boundedY = clamp(evt.unprojectedPoint.y, fov.bottom, fov.top);
```

Comment:
`bound` is already past tense :wink: - maybe just use the same names as in `useMoveCameraTo`: `xBound` and `yBound`?

Reply:
Can't we say that the FOV bounds the coordinates ? Therefore, the coordinate is bounded ? :thinking: 

To me, `xBound` means the value is the limit in x whereas here, it stores the x value limited by the FOV.

So, the meanings are not the same but I am open to other namings !

Reply:
Oh apologies, `bounded` is indeed the paste tense of "to bound" 😅 and I get what you mean about the two different meanings of the variables. 👍